src/Options/Generic.hs

```diff
@@ -1133,8 +1134,20 @@ getWithHelp
     -- ^ Program description
     -> io (a, io ())
     -- ^ (options, io action to print help message)
-getWithHelp desc = do
-  a <- getRecordWith header mempty
+getWithHelp desc = getRecordWithHelp desc mempty
+
+-- | Marshal any value that implements `ParseRecord` from the commmand line
+-- alongside an io action that prints the help message.
+getRecordWithHelp
```

Comment:
For consistency, this should probably be called `getWithHelpWith`, even if that is kind of an awkward name